components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -332,7 +329,8 @@ class RewardsServiceImpl : public RewardsService,
   void SetCatalogIssuers(const std::string& json) override;
   void AdSustained(const std::string& json) override;
   void SetConfirmationsIsReady(const bool is_ready) override;
-  void GetAdsNotificationsHistory() override;
+  void GetAdsNotificationsHistory(
+      brave_rewards::AdsNotificationsHistoryCallback callback) override;
   void ConfirmationsTransactionHistoryDidChange() override;
```

Comment:
per comment below, can we also send the data here so we don't have to make another call to get it?

Reply:
let's do this in a follow up, cc @tmancey